angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -120,7 +120,7 @@ <h4 class='header'>{{domainResultsLabel}}</h4>
 
     </div>
     <div>
-      <h4 ng-if="webSearchUrl && !googleEmptyResults"><a ng-href="{{webSearchUrl + searchText}}">View more results for {{searchText}} on {{domainResultsLabel}}</a></h4>
+      <h4 ng-if="webSearchUrl && !googleEmptyResults"><a ng-href="{{webSearchUrl + searchText}}" target=_blank>View more results for {{searchText}} on {{domainResultsLabel}}</a></h4>
```

Comment:
This will almost certainly work, but I'd have done ` target="_blank"` so that the attribute value is quoted in the same way that e.g. the `ng-href` attribute value is quoted. Consistency?